update/UpdateProvider.cfc

```diff
@@ -23,6 +24,11 @@
 		directoryCreate(variables.artDirectory);
 	variables.extDirectory="/var/www/extension/extension/"; // TODO make more dynamic
 
+	private function logger( string text, string stack="", type="info" ){
+		var log = arguments.text & chr(13) & chr(10) & callstackGet('string') & chr(13) & chr(10) & arguments.stack;
+		WriteLog( text=log, type=arguments.type, log=variables.providerLog );

```

Comment:
you can also feed an exception into log with the attribute "exception"

Reply:
ah yeah, does the sentry extension do something sexy with that?